content/klc/S4.5.adoc

```diff
@@ -2,10 +2,14 @@
 brief = "Pins not connected on the footprint may be omitted from the symbol"
 +++
 
-Often a component has pins that are not physically connected, and in such cases, unconnected pins may be omitted from the schematic symbol.
+Often a component has pins that are not physically connected.
+Even these pins should be included in the symbol. Refer to the link:/libraries/klc/S4.6[requirements for hidden pins].
+
+However in some cases unconnected pins may be omitted from the schematic symbol.
 
 . Pins may only be omitted if they are never to be connected _under any circumstances_
+. Unconnected pins may only be omitted if including them would make the symbol unnecessarily large.
 . If pins are designated `NC` but the datasheet specifies that these must be pulled to ground or otherwise connected to a specific net, then these **must** be shown on the schematic symbol
-. If the pin-count in the schematic does not match the pad-count on the footprint, the footprint filter must include the pad-count of the footprint (see also link:/libraries/klc/S5.2[requirements for footprint filters])
-* `SOT?23?5`
-* `SOIC?8`
+. The footprint filter must include the pad-count of the footprint if the pin-count in the schematic does not match the pad-count of the footprint (see also link:/libraries/klc/S5.2[requirements for footprint filters])
```

Comment:
Sounds better to me if `in the schematic` changes to `of the symbol`. Also, why hyphenate `pincount`?

Reply:
found an even better wording